### PR TITLE
[Alicloud BOSH Docs] Add Bosh Docs for Alibaba Cloud CPI

### PR DESCRIPTION
**Pull request by xiaozhu36**
The PR add bosh docs which applies to alibaba cloud CPI configuration and properties.
The PR includes the following files:
 
* init-alicloud.md: creating alicloud environment details before deploying BOSH
* alicloud-cpi.md: alicloud bosh CPI details

### CODE REVIEW

File: alicloud-cpi.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+    security_group_ids: ["sg-0xiecbyds24g3b9dn14p"]
+```
+
+Example of vip network:
+
+```yaml
+networks:
+- name: default
+  type: vip
+  ip: 47.47.47.47
```
Comment (cppforlife) [inline alicloud-cpi.html.md.erb]: `ip` is not specified in this section.

Comment (xiaozhu36) [reply alicloud-cpi.html.md.erb]: Hi @cppforlife thanks for your feedback and I have modify it.

File: content/init-alicloud.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        -v zone=us-east-1a \
+        -v vswitch_id=vsw-rj9rio... \
+        -v security_group_id=sg-rj9dtcbw... \
+        -v key_pair_name=bosh \
+        -v private_key=~/Downloads/bosh.pem
+    ```
+
+    If running above commands outside of an Alicloud VPC, refer to [Exposing environment on a public IP](init-external-ip.md) for additional CLI flags.
+
+    See [Alicloud CPI errors](alicloud-cpi-errors.md) for list of common errors and resolutions.
```
Comment (pivotal-dale-wick) [inline content/init-alicloud.md]: The file `alicloud-cpi-errors.md` is missing from this PR.  Please fix, and update the PR.  Thanks.

File: mkdocs.yml [hunk 3]
```diff
@@ -205,6 +206,8 @@ pages:
     - Deploying BOSH with create-env: init.md
 
   - "--- Cloud Providers": divider.md
+  - Alibaba Cloud:
+    - Usage: alicloud-cpi.md
```
Comment (pivotal-dale-wick) [inline mkdocs.yml]: Please add all new pages here.

**Comment by jfmyers9** [comment]
Hey @xiaozhu36,

Have you had a chance to take a look at the changes we suggested? Let us know if you have any questions.

Best,

@jfmyers9 && @belinda-liu, CF BOSH

**Comment by jfmyers9** [comment]
Hi all,

We are going to close this issue due to inactivity. Please feel free to resubmit the PR if you are still looking to get Alicloud documentation into bosh.io.

Best,

@jfmyers9 && @belinda-liu, CF BOSH

**Pull request closed without merging**